source/languages/en/riak/dev/references/protocol-buffers/store-object.md

```diff
@@ -69,7 +69,7 @@ a special value denoting `one` (`4294967295-1`), `quorum`
 
 Parameter | Description
 :---------|:-----------
-`key` | The key to create/update. If not specified, Riak will generate a random key and return that key as part of the requests's
+`key` | The key to create/update. If not specified, Riak will generate a random key and return that key as part of the requests'
```

Comment:
I went back to the source to get context on this whole segment (not your fix) because it looked... weird. I think what it should really say is, "response to that request." instead of "requests' ". If you feel like fixing that, awesome! If not, that's OK.